fvtest/lljbtest/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+###############################################################################
+# Copyright (c) 2020, 2020 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+find_package(LLVM 8 REQUIRED CONFIG)
+
+# we need clang to generate IR from C++
+find_program(CLANG_EXECUTABLE  NAMES clang)
+
+if (NOT CLANG_EXECUTABLE)
+    message(SEND_ERROR "Could not find clang")
+endif()
+
+# Generate llvm bitcode files from C++ using clang
+function(generate_module_from_cpp src)
+    add_custom_command(
+        OUTPUT
+            ${CMAKE_CURRENT_BINARY_DIR}/${src}.ll
+        COMMAND
+            ${CLANG_EXECUTABLE} -S -emit-llvm -std=c++0x -O0 -o ${CMAKE_CURRENT_BINARY_DIR}/${src}.ll ${CMAKE_CURRENT_SOURCE_DIR}/${src}.cpp
+        MAIN_DEPENDENCY
+            ${CMAKE_CURRENT_SOURCE_DIR}/${src}.cpp
+    )
+    add_custom_target(generate_module_from_cpp_${src} ALL
+        DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${src}.ll
+    )
+
+endfunction(generate_module_from_cpp)
+
+
+
+add_executable(lljb_run
+	lljb_run.cpp
+)
+
+target_include_directories(lljb_run
+    PRIVATE
+        ${CMAKE_SOURCE_DIR}/jitbuilder/
+        ${CMAKE_SOURCE_DIR}/compiler/
+        ${CMAKE_SOURCE_DIR}/omr/
```

Comment:
These three paths should be rooted by `${omr_SOURCE_DIR}`, so that the paths are still correct when OMR is a subdirectory of a bigger project.

Reply:
Actually, these shouldn't be needed, since lljb should provide all necessary include directories.